README.md

```diff
@@ -98,7 +98,18 @@ Tests will be performed on each of the methods (e.g. feature extraction) by comp
 7. 3.22; and
 8. latest version.
 
-## Manual testing
+## Local testing
+To perform local testing the 'run-docker-tests.sh' can be used. The shell script can be used as follows:
+1. Open your terminal/console;
+2. Go to the root directory of the plugin;
+3. Type "./run-docker-test.sh" and press enter;
+4. The result should be as follows:
+
+![segment_center](https://github.com/kartoza/stream_feature_extractor/blob/develop/help/source/examples/local_testing.png)
+
+5. If there is any errors the user will need to investigate.
```

Comment:
We should remove this line, and instead explain what is need for the tests to run successfully.